eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -91,8 +91,7 @@ public function load(array $configs, ContainerBuilder $container)
 
         $configuration = $this->getConfiguration($configs, $container);
 
-        $environment = $container->getParameter('kernel.environment');
-        if (in_array($environment, ['behat', 'test'])) {
+        if ($this->shouldLoadTestServices($container)) {
```

Comment:
Now that I see it, I think it should be a separate bundle, not a part of Core ;)